src/Compute/Compute/help/Get-AzVMSqlServerExtension.md

```diff
@@ -42,7 +42,7 @@ This command gets the settings of the SQL Server extension on a virtual machine
 
 ### Example 2: Get the settings by using the pipeline
 ```powershell
-Get-AzVM -ServiceName "Service08" -Name "ContosoVM22" | Get-AzVMSqlServerExtension
```

Comment:
Can Get-AzVM work without -ResourceGroupName? Is it better to edit it to 'Get-AzVM -ResourceGroupName "ResourceGroup11" -Name "ContosoVM07" | Get-AzVMSqlServerExtension'?

Reply:
It works and it also makes sense by using -ResourceGroupName, so I will add it.

![image](https://user-images.githubusercontent.com/95342181/158984565-c7451843-b91f-477b-97f9-c61694c9870c.png)